Sequence of Tenses/maxi.md

```diff
@@ -0,0 +1,39 @@
+# Please translate the following sentences into English
+
+1. На прошлой неделе Сергей заметил, что его коллега по проекту пользуется готовыми снипетами, что ускоряло его работу, но видимо он передумал и решил писать код по старинке.
+
+On the previous week, Sergiy noticed, that his colleague on the project using the prepared snippet of code, that increased the speed of work, but he changed the mind and decided writing the code as usual 
+
+2. Тим лид признался что всем придется писать юнит тесты по своему функционалу и то что они делали до этого придется переделать с ноля, так как отдел контроля качества забраковал первичный вариант проекта.
+
+Team lead admitted that everyone must write the unit tests for functionality that they did,  and everything that previously had done need to be reworked from scratch, because QA discards the very first variant of the project. 
+
+3. Кен сказал что все что было сделано, будет переделано, а то что должно было быть сделано, будет отложено на потом.
+
+Ken said, everything that had been done would be redone and everything that had supposed to be done, would be postponed.
+
+4. Роман сказал что должен был явиться на работу к 10 часам, но опоздал потому что проспал и не смог быстро вызвать такси.
+
+Roman said, that he must be at work at 10 o'clock, but he was late because he overslept, and he was not able to call the taxi.
+
+5. В прошлом месяце я сказал ей, что нам придется написать техническую документацию к проекту и думал что она приступит к работе сразу же, но она еще даже не начала.
+
+Last month, I told her that we would have to write tech doc for the project, and I thought she started immediately, but she hasn't started 
+
+6. Сергей сказал Владу, что необходимо будет решить пару организационных вопросов перед пересмотром повышения зарплаты, на что тот ему ответил, что он будет готов обсудить любые варианты.
+
+Sergey said, that would require to solve several organizational issues before promotion, and he replied,  that he would be ready to discuss any variants 
```

Comment:
say is used without the object in case you need an object you are supposedto use tell + object = Sergey told Vlad that
it would be necessary to resolve